### PR TITLE
Logging In Quickbyte (Branching off of #175)

### PR DESCRIPTION
**Pull request by rysc3**
This MR is branching off of #175, so the link is valid.

### CODE REVIEW

**Comment by rysc3** [comment]
Maybe we should throw the date we last made sure the quickbyte was accurate there? Obviously they can check when the file was last edited but this could more so represent the last date we checked that the instructions actually work, not necessarily edited the file?

**Comment by rysc3** [comment]
Changed verbiage to "validated" as discussed in res. support meeting 5/21/24

File: logging_in.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
 # Logging in
 
-In order to use log in to CARC systems and start computing you will need a terminal that can log in to remote systems using Secure Shell (SSH). If you are using a Linux or Mac machine you are in luck as they come bundled with a terminal that is ready to use. The terminal application packaged with your OS is great, but if you would like something with a bit more customization, utility, and flexibility there are other free options available. Below are a couple of options that CARC recommends. Note that iTerm2 is Mac only.
+In order to use log in to CARC systems and start computing you will need a terminal that can log in to remote systems using Secure Shell (SSH). If you are using a Linux or Mac machine you are in luck as they come bundled with a terminal that is ready to use. The terminal application packaged with your OS is great, but if you would like something with a bit more customization, utility, and flexibility there are other free options available. Below are a couple of options that CARC recommends. Note that iterm2 is only available on Mac, and MobaXterm is only available on Windows.
```
Comment (gmfricke) [inline logging_in.md:3]: Check grammar here

**Review by gmfricke — CHANGES_REQUESTED** [review]
Grammar - "in order "

**Review by gmfricke — APPROVED** [review]

**Pull request merged**